src/views/main/components/ChartTabs.js

```diff
@@ -0,0 +1,201 @@
+import React, { Component } from 'react';
+import Chart from 'chart.js';
+import Tabs, { Tab } from 'react-awesome-tabs';
+import 'react-awesome-tabs/src/sass/react-awesome-tabs.scss';
+import { AirQualityColors, indexToLevel } from '../../../config/AirQuality';
+
+class ChartTabs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      stationDetal: props.stationDetal,
+      activeTab: 0,
+    };
+  }
+
+  handleTabSwitch(active) {
+    this.setState({ activeTab: active });
+  }
+
+  mapStateToBarColor(airQialityIndex) {
+		return AirQualityColors[indexToLevel(airQialityIndex)];
+	}
+
+  barChart(handler, labels, chartData, dataColor) {
+    new Chart(handler.getContext('2d'), {
+      type: 'bar',
+      data: {
+        labels: labels,
+        datasets: [{
+          label: null,
+          data: chartData,
+          backgroundColor: dataColor,
+          borderColor: 'rgba(255, 255, 255, 0)',
+          borderWidth: 1
+        }]
+      },
+      options: {
+        scales: {
+          xAxes: [{
+            ticks: {
+              display: false,
+            },
+            gridLines: {
+              display: false,
+            },
+            categoryPercentage: 0.9,
+            barPercentage: 1.0
+          }],
+          yAxes: [{
+            ticks: {
+              beginAtZero: true,
+              stepSize: 10,
+              suggestedMax: 60,
+            },
+          }]
+        },
+        legend: {
+          display: false
+        },
+        tooltips: {
+          callbacks: {
+            label: function (tooltipItem, data) {
+              return "Quality index: " + tooltipItem.yLabel;
+            }
+          }
+        },
+      }
+    });
+  }
+
+  makeAirQualityChart() {
+    let handler = document.getElementById('airQualityChart');
+    if (!handler) return;
+    let airQualityData = this.state.stationDetal.sensors.airQuality.values;
+
+    let labels = airQualityData.map(data => {
+      let timestamp = new Date(data.timestamp);
+      return 'Czas: ' + timestamp.getHours() + ':00';
+    });
+
+    let chartData = airQualityData.map(data => { return data.state });
+
+    let dataColor = airQualityData.map(data => {
+      return this.mapStateToBarColor(data.state);
+    });
+
+    this.barChart(handler, labels, chartData, dataColor);
```

Comment:
Nie wiem, czy pokazywanie tego kwadratu z kolorem jest konieczne w każdej sytuacji. Na PM fajnie to wygląda, bo lepiej widać, ale na wykresie słupkowym wydaje mi się ten kolorek trochę niepotrzebny. Tam kolumna już ma swój kolor i tak... Ale to szczegół, daję komentarz tylko dla zastanowienia. No i nie wiem, czy nie pokazywanie tego nie byłoby za bardzo kłopotliwe w implementacji, więc pewnie lepiej zostawić.